From 381ab833b06159a9c26ab971cdeed09ff41c4f22 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Wed, 1 Aug 2018 11:33:30 +0800
Subject: [PATCH 27/32] sunxi: add DDC_EN pin for Pine H64

Some versions of Pine H64 has a DDC_EN pin which gates the I/O of the
HDMI DDC pins.

Add the pin in config for Pine H64.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 configs/pine_h64_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/pine_h64_defconfig b/configs/pine_h64_defconfig
index e9596c0284..80f172090d 100644
--- a/configs/pine_h64_defconfig
+++ b/configs/pine_h64_defconfig
@@ -3,6 +3,7 @@ CONFIG_ARCH_SUNXI=y
 CONFIG_MACH_SUN50I_H6=y
 CONFIG_DRAM_ODT_EN=y
 CONFIG_MMC0_CD_PIN="PF6"
+CONFIG_HDMI_DDC_EN="PH2"
 CONFIG_MMC_SUNXI_SLOT_EXTRA=2
 # CONFIG_PSCI_RESET is not set
 CONFIG_DEFAULT_DEVICE_TREE="sun50i-h6-pine-h64"
-- 
2.18.0

